File: uxmobile/src/main/java/sk/brecka/uxmobile/model/event/ClickEvent.java
package sk.brecka.uxmobile.model.event;

import sk.brecka.uxmobile.model.ViewEnum;

/**
 * Created by matej on 4.10.2017.
 */

public class ClickEvent extends ViewEvent {

    public ClickEvent(long startTime, double x, double y, ViewEnum viewEnum, String viewText, String viewInfo) {
        super(startTime, x, y, viewEnum, viewText, viewInfo);
    }

    @Override
    protected String getType() {
        return TYPE_CLICK;
    }

}

File: uxmobile/src/main/java/sk/uxtweak/uxmobile/model/event/ClickEvent.kt
package sk.uxtweak.uxmobile.model.event

import sk.uxtweak.uxmobile.model.ViewEnum

class ClickEvent(
    startTime: Long,
    x: Float,
    y: Float,
    viewEnum: ViewEnum,
    viewText: String,
    viewValue: String
) : ViewEvent(startTime, x, y, viewEnum, viewText, viewValue) {
    override fun getType() = TYPE_CLICK
}
